src/megaclient.cpp

```diff
@@ -9359,7 +9359,7 @@ void MegaClient::notifynode(Node* n)
                     n->localnode->deleted = n->changed.removed;
                 }
 
-                if (!n->changed.removed && n->changed.parent)
+                if (!n->changed.removed && n->changed.newnode)
```

Comment:
I think it could also be a new parent. Depending on the nature of the movement, it could result in a `copy`+`delete` or in a `move`. So...
```suggestion
                if (!n->changed.removed && (n->changed.newnode || n->changed.parent))
```